packages/SwingSet/src/kernel/kernel.js

```diff
@@ -1325,7 +1362,9 @@ export default function buildKernel(
     const { computrons, meterID } = crankResults;
     if (computrons) {
       assert.typeof(computrons, 'bigint');
-      policyInput[1].computrons = BigInt(computrons);
+      if (policyInput[0] !== 'cleanup') {
+        policyInput[1].computrons = BigInt(computrons);
+      }
```

Comment:
What would be the harm in keeping the code that sets `computrons` rather than special-casing cleanup here?

Reply:
hm, I don't remember my exact reasoning.. I think I was uncomfortable with providing data that we know is useless, except that in this case we know we didn't do any deliveries, so we know for sure that no vat worker has consumed any computrons, so "0" is in fact accurate, and not just a lack of data

I'll switch it back.

Reply:
oh and the `if (computrons)` guard means that code won't get executed anyways, because the `cleanup-terminate-vat` handler always sets `computrons = 0n`

I think I didn't re-read that code carefully enough before commiting it..